external/docker/spark-mesos/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apt-get update && \
     apt-get install -y python libnss3 openjdk-7-jre-headless curl
 
 RUN mkdir /opt/spark && \
-    curl http://www.apache.org/dyn/closer.lua/spark/spark-1.4.0/spark-1.4.0-bin-hadoop2.4.tgz \
+    curl https://www.apache.org/dyn/closer.lua/spark/spark-1.4.0/spark-1.4.0-bin-hadoop2.4.tgz \
```

Comment:
BTW this link is dead; this whole Dockerfile looks obsolete anyway

Reply:
yea, this isn't going to work...

Reply:
Then, shall we remove this `external/docker` since we have K8S?

Reply:
I'd do it separately, but, yeah I only see it referenced in `conf/docker.properties.template`, which itself isn't referenced, and this whole thing looks very out of date anyway.

Reply:
Thanks. Could you rebase this?